core/util/inAppLink.jsx

```diff
@@ -1,9 +1,40 @@
+function openUrl(url) {
+  SafariViewController.isAvailable(function (available) {
```

Comment:
@jbaxleyiii I did think about this, but I saw this on their end:

https://github.com/EddyVerbruggen/cordova-plugin-safariviewcontroller/commit/19266d46347ef45f5cdf2955a00d5f05a128ca57

You are probably right to be safe though, could possibly not load or something.